server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -506,14 +506,24 @@ public void reportShuffleResult(
         "appId[" + appId + "], shuffleId[" + shuffleId + "], taskAttemptId[" + taskAttemptId + "]";
 
     try {
+      int expected =
```

Comment:
Maybe `expectedNum` and `updatedNum`?

Reply:
Needn't. The type has indicated everything.